src/sentry/tasks/derive_code_mappings.py

```diff
@@ -26,41 +22,29 @@
 logger = logging.getLogger("sentry.tasks.derive_code_mappings")
 
 
-@instrumented_task(
-    name="sentry.tasks.derive_code_mappings.process_organizations",
-    queue="derive_code_mappings",
-    max_retries=0,
-)  # type: ignore
-def derive_missing_codemappings(dry_run=False) -> None:
-    """
-    Queue tasks to process each organization to derive missing codemappings.
-    """
-    organizations = Organization.objects.filter(status=OrganizationStatus.ACTIVE)
-    for _, organization in enumerate(
-        RangeQuerySetWrapper(organizations, step=1000, result_value_getter=lambda item: item.id)
-    ):
-        if not features.has("organizations:derive-code-mappings", organization):
-            continue
-
-        # Create a celery task per organization
-        derive_code_mappings.delay(organization.id)
-
-
 @instrumented_task(  # type: ignore
     name="sentry.tasks.derive_code_mappings.derive_code_mappings",
     queue="derive_code_mappings",
     max_retries=0,  # if we don't backfill it this time, we'll get it the next time
 )
-def derive_code_mappings(organization_id: int, dry_run=False) -> None:
+def derive_code_mappings(
+    organization_id: int,
+    project_id: int,
```

Comment:
Having just the project_id might be sufficient since a project has a reference to the organization.
https://github.com/getsentry/sentry/blob/a614d82637d6cb3d20d8c95da487e3c04c715732/src/sentry/models/project.py#L106-L119